R/RunDiagnostics.R

```diff
@@ -49,6 +49,10 @@
 #' @param runIncidenceRate            Generate and export the cohort incidence  rates?
 #' @param runCohortOverlap            Generate and export the cohort overlap?
 #' @param runCohortCharacterization   Generate and export the cohort characterization?
+#' @param covariateSettings           (OPTIONAL) Either an object of type \code{covariateSettings} as created using one of
```

Comment:
This argument is not optional, it must be provided. Luckily, if the user doesn't specify it , a default value is used. (The same is true for other arguments like `runCohortOverlap`.)

Please don't document default values of arguments. These will be automatically shown in the help file, where people can see it defaults to `FeatureExtraction::createDefaultCovariateSettings()`.

Reply:
> Please don't document default values of arguments. These will be automatically shown in the help file, where people can see it defaults to FeatureExtraction::createDefaultCovariateSettings().

Thank you!